### PR TITLE
Update to new config format

### PR DESCRIPTION
**Pull request by qm3ster**

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -5,9 +5,11 @@ import pkg from './package.json';
...
 	{
-		entry: 'src/main.js',
-		dest: pkg.browser,
-		format: 'umd',
+		input: 'src/main.js',
+		output: {
+			file: pkg.browser,
+			format: 'umd'
+		}
 		moduleName: 'howLongUntilLunch',
```
Comment (riccardoscalco) [inline rollup.config.js]: According to [here](https://gist.github.com/Rich-Harris/d472c50732dab03efeb37472b08a3f32), field `moduleName` also changed to `name`.

Comment (qm3ster) [reply rollup.config.js]: Thank you, good catch.
Did it produce a depreciation message?

Comment (riccardoscalco) [reply rollup.config.js]: Yes, in the message `moduleName` was listed among the other changed fields.

**Comment by rmariuzzo** [comment]
Thank you @eventualbuddha !

**Comment by lukepetrolekas** [comment]
Please bump the version number for rollup to 0.50 in package.json accordingly so other users don't get confused like I did :)

**Pull request merged**